### ISSUE TITLE
environment.url referencing step output causing errors.

### ISSUE THREAD

**Issue by flcdrg** [rhysd/actionlint#30]
I have a workflow similar to the following (code edited slightly):

```yaml
jobs:
  myjob:
    environment:
      name: env-name
      url: ${{ steps.thing.outputs.app-url }}

  steps:
      - name: 'Run Azure Functions Action'
        uses: Azure/functions-action@v1.4.0
        id: thing
        with:
          app-name: 'my-function-app'
          package: my.zip
```

The linter is throwing an error which seems to be invalid to me.

```
.github/workflows/thing.yml:86:16: property "thing" is not defined in object type {} [expression]
   |
86 |       url: ${{ steps.thing.outputs.app-url }}
   |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by rhysd** [comment]
Thank you for reporting. This must be a bug.

**Comment by rhysd** [comment]
This was released at [v1.6.1](https://github.com/rhysd/actionlint/releases/tag/v1.6.1).
